conf/modules_esmfold.config

```diff
@@ -14,11 +14,19 @@ process {
     withName: 'RUN_ESMFOLD' {
         ext.args = {params.use_gpu ? '' : '--cpu-only'}
         publishDir = [
+            [
                 path: { "${params.outdir}/esmfold" },
                 mode: 'copy',
                 saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
-                pattern: '*.*'
+                pattern: '*'
```

Comment:
Why this pattern is changed?

Reply:
Ups I think I was playing around and it's a leftover - fixed!